### PR TITLE
[RPW]Fix Diffusion Profile Volume Component inspector when opened wit…

### PR DESCRIPTION
**Pull request by alex-vazquez**
---
### Purpose of this PR
The OnEnable method was being called, before the inspector was being set. The OnEnable is being called manually by the VolumeComponentListEditor

---
### Testing status
- Check the DiffusionProfile editor works as expected.

### CODE REVIEW

**Review by adrien-de-tocqueville — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/CHANGELOG.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixing missing doc API for RTAS Debug display.
 - Fixed AO dissapearing when DRS would be turned off through a camera, while hardware drs is active in DX12 or Vulkan (case 1383093).
 - Fixed misc shader warnings.
+- Fixed crash opeing Volume Component with Diffusion Profile.
```
Comment (remi-chapelain) [inline com.unity.render-pipelines.high-definition/CHANGELOG.md:84]: minor typo here

**Review by remi-chapelain — APPROVED** [review]
Quickly checked to see if no issues arised, LGTM ✔️

File: com.unity.render-pipelines.high-definition/CHANGELOG.md (hunk 1 above)
Comment (alex-vazquez) [inline com.unity.render-pipelines.high-definition/CHANGELOG.md:84]:
```suggestion
- Fixed NullReferenceException when opeining a Volume Component with a Diffusion Profile with any inspector.
```

File: com.unity.render-pipelines.high-definition/CHANGELOG.md [hunk 2]
```diff
@@ -81,6 +81,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixing missing doc API for RTAS Debug display.
 - Fixed AO dissapearing when DRS would be turned off through a camera, while hardware drs is active in DX12 or Vulkan (case 1383093).
 - Fixed misc shader warnings.
+- Fixed NullReferenceException when opeining a Volume Component with a Diffusion Profile with any inspector.
```
Comment (adrien-de-tocqueville) [inline com.unity.render-pipelines.high-definition/CHANGELOG.md:84]: lol minor typo here

**Comment by alex-vazquez** [comment]
Safety Net failing because : `RenderSettingsConverter.cs(264,22): error CS0618: 'RenderingPath.DeferredLighting' is obsolete: 'DeferredLighting has been removed. Use DeferredShading, Forward or HDRP/URP instead.` Merging.

**Pull request merged**